package.json

```diff
@@ -54,5 +54,10 @@
     "ejs": "^3.1.8",
     "fork-ts-checker-webpack-plugin": "^6.0.4",
     "globby": "^11.0.2"
+  },
+  "dependencies": {
```

Comment:
these can be removed, right?
These seem to already be defined in `packages/workflow/package.json`

Reply:
Ahh, yeah. This likely happened in one of the 5 times I had to rebase to remove Brian's email.